types/edit-json-file/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for edit-json-file 1.4
+// Project: https://github.com/IonicaBizau/edit-json-file#readme
+// Definitions by: Twixes <https://github.com/Twixes>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types= "node" />
+
+import { NoParamCallback } from 'fs';
+
+declare namespace editJsonFile {
+    /** JSON file editor options. */
+    interface Options {
+        stringify_width?: number;
+        stringify_fn?: (data: object) => string;
+        stringify_eol?: boolean;
+        autosave?: boolean;
+    }
+
+    /** JSON file editor. */
+    class JsonEditor {
```

Comment:
I don't think this class is exported by the underlying library?  E.g. you can't write
```js
const editJsonFile = require("edit-json-file");
const editor = new editJsonFile.JsonEditor("path");
```

If you're just trying to express the shape of the return type, I think you can just change `class` to `interface` and drop the constructor.

Reply:
You're right, JsonEditor is not exported. Changed that.